server/backend/mlx.py

```
@@ -268,7 +152,7 @@ def _store_response(
         except Exception:
             pass
     try:
-        _responses[response_id] = resp
+        is_answerresponses[response_id] = resp
```

Ruff (0.15.13)

[error] Undefined name `is_answerresponses`

(F821)